app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/CustomScriptRow/index.jsx

```diff
@@ -278,8 +279,10 @@ const CustomScriptRow = ({
         type: GLOBAL_CONSTANTS.SCRIPT_LOAD_TIME.AFTER_PAGE_LOAD,
       },
       {
-        title: "Before Page Load",
-        type: GLOBAL_CONSTANTS.SCRIPT_LOAD_TIME.BEFORE_PAGE_LOAD,
+        title: isExtensionManifestVersion3() ? "As Soon As Possible" : "Before Page Load",
+        type: isExtensionManifestVersion3()
+          ? GLOBAL_CONSTANTS.SCRIPT_LOAD_TIME.AS_SOON_AS_POSSIBLE
+          : GLOBAL_CONSTANTS.SCRIPT_LOAD_TIME.BEFORE_PAGE_LOAD,
```

Comment:
will this not cause errors like `the constant not being identified` inside mv2? should also add a check there to start ignoring this new loadTime constant, right?